data/json/recipes/ammo/pistol.json

```diff
@@ -1,5 +1,41 @@
 [
   {
+    "result": "36navy",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_PISTOL",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "skills_required": [ "gun", 1 ],
+    "time": 15000,
+    "book_learn": [ [ "manual_pistol", 3 ], [ "recipe_bullets", 3 ] ],
+    "using": [ [ "ammo_bullet", 30 ] ],
+    "qualities": [ { "id": "CUT", "level": 1, "amount": 1 } ],
+    "components": [
+      [ [ "gunpowder", 100 ], [ "chem_black_powder", 200 ] ],
```

Comment:
New recipes should be per-charge, especially for this sort of ammunition. Check out the definitions for 9mm for a relevant example